ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -55,6 +65,13 @@ SettingsContentBase {
                 root.rootStore.backButtonName = qsTr("Wallet")
                 root.sectionTitle = qsTr("Networks")
             }
+            if(currentIndex == root.editNetworksViewIndex) {
+                root.rootStore.backButtonName = qsTr("Networks")
+                root.sectionTitle = qsTr("Edit %1").arg(!!editNetwork.combinedNetwork.prod && !!editNetwork.combinedNetwork.prod.chainName ? editNetwork.combinedNetwork.prod.chainName: "")
```

Comment:
layer is different, and I agree with you to combine it. however we will only ever have to types prod and test networks for the same chain, so that shouldn't be a problem, but I see your suggestion and I think its a good idea too, I am going to try it out and see how it simplifies things :)